src/frequenz/client/dispatch/test/__init__.py

```diff
@@ -0,0 +1,10 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Helpful utilities for testing with the dispatch client."""
+
+from .sampler import Sampler
+from .service import MockClient as Client
```

Comment:
I wouldn't rename this, I think it is still more clear to keep the `Mock` prefix even if it is in the `test` package.